typings/sql-bindings.d.ts

```diff
@@ -0,0 +1,102 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+declare module 'sql-bindings' {
```

Comment:
Why is this here? The extension shouldn't need to have this.

Reply:
Yeah, I can delete the typing files as well now.